package.json

```diff
@@ -28,12 +28,14 @@
     "code-push-plugin-testing-framework": "file:./code-push-plugin-testing-framework",
     "del": "latest",
     "express": "latest",
-    "gulp-typescript": "2.12.2",
     "gulp-insert": "latest",
     "gulp-tslint": "latest",
+    "gulp-typescript": "2.12.2",
     "mkdirp": "latest",
     "q": "^1.4.1",
-    "run-sequence": "latest"
+    "run-sequence": "latest",
+    "tslint": "^4.3.1",
+    "typescript": "^2.1.5"
```

Comment:
Nothing special, just wanted to avoid npm _UNMET PEER DEPENDENCY_ warnings:

>npm WARN gulp-tslint@7.1.0 requires a peer of tslint@>=4.0.0-dev but none was installed.
>npm WARN tslint@4.4.2 requires a peer of typescript@>=2.0.0 but none was installed